flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/DistinctAggregateTest.scala

```diff
@@ -117,6 +117,11 @@ class DistinctAggregateTest(
     util.verifyPlan(sqlQuery)
   }
 
+  @Test
+  def testTwoDistinctAggregateWithNonDistinctAgg(): Unit = {
+    util.verifyPlan("SELECT c, SUM(DISTINCT a), SUM(a), COUNT(DISTINCT b) FROM MyTable GROUP BY c")
```

Comment:
What's purpose of this test? Will it fail without the fix?

Reply:
yes, to verify the bug